src/components/Nav/ConnectionMenu.vue

```diff
@@ -175,7 +177,12 @@ export default {
           <span v-else>Couldn't connect</span>
           to
           <span class="font-weight-bold">
-            <span v-if="isCloud" class="primary--text">Prefect Cloud</span>
+            <span v-if="isCloud" class="primary--text"
+              >Prefect Cloud
+              <span v-if="apiMode === 'maintenance'" class="primary--text">
+                - In Maintenance</span
```

Comment:
I think this would be better to leave the title as-is and to add some text below, for example:
```
Prefect Cloud

Prefect Cloud is currently undergoing maintenance; you may see minor service disruptions as a result. Check out the <ExternalLink href="https://prefect.status.io/">Prefect Cloud status page</ExternalLink> for information about upcoming maintenance windows and the current uptime of Prefect Cloud services.

```

Reply:
The wording is there already.  i added the extra 'In Maintenance' in the title as it's easier to see at a glance why the indicator is yellow. 
<img width="549" alt="Screen Shot 2021-05-10 at 10 54 40 AM" src="https://user-images.githubusercontent.com/40272060/117679464-4cf43300-b17e-11eb-9c38-b99dad2ab38d.png">
<img width="578" alt="Screen Shot 2021-05-10 at 10 53 58 AM" src="https://user-images.githubusercontent.com/40272060/117679479-4fef2380-b17e-11eb-9f14-1da22f7694bb.png">